crates/libduckdb-sys/upgrade.sh

```diff
@@ -14,11 +14,11 @@ export DUCKDB_LIB_DIR="$SCRIPT_DIR/duckdb"
 # Download and extract amalgamation
 DUCKDB_VERSION=v1.2.0
 git submodule update --init --checkout
-cd "$SCRIPT_DIR/duckdb-sources"
-git fetch
-git checkout "$DUCKDB_VERSION"
+# cd "$SCRIPT_DIR/duckdb-sources"
+# git fetch
+# git checkout "$DUCKDB_VERSION"
 cd "$SCRIPT_DIR"
-python3 "$SCRIPT_DIR/update_sources.py"
+DUCKDB_EXPLICIT_VERSION=v1.2.0 python3 "$SCRIPT_DIR/update_sources.py"
```

Comment:
This allows building configured submodule version as v1.2.0 w/o requiring to have `v1.2.0` tag (in case we will need more updates)